src/auto-merge.js

```diff
@@ -8,7 +8,7 @@ const repoWhitelist = [
   'testing-stuff'
 ]
 
-module.exports = (robot) => {
+export default (robot) => {
```

Comment:
In general I'm not a fan of exporting defaults because it make it harder for autocompleters to find what I want to import but it seems like the reason was because less code needed to be changed. Cool

Reply:
i also have experienced this but it always seemed to me like a bad reason to change the code. i wish the auto importers would just be smart enough to look for filenames.